### ISSUE TITLE
freedraw eraser

### ISSUE THREAD

**Issue by dwelle** [excalidraw/excalidraw#3682]
For other shapes an eraser does not make much sense (at least not with respect to their implementation), but for a freedraw it kinda does.

**Comment by 3nzoFerrari** [comment]
Support for stylus i.e. Wacom Bamboo Inc would be nice

**Comment by TheMathGuyd** [comment]
Using Mac and iPad so it’s goodnotes for me. I think there are many in the same boat as you and I. It would be awesome to have a web-based open-source alternative. 

excalidraw has most features matched with goodnotes except for eraser and highlighter tools.

**Comment by oak86** [comment]
Yeah, I definitely need an eraser feature. It would make a huge difference as I depend on it in my current workflow on my iPad

**Comment by melsophos** [comment]
Excalidraw definitely needs an eraser for `freedraw` which would allow to erase part of a stroke.

However, I think it would also make sense to have a stroke eraser which deletes a full continuous stroke. This can be useful to quickly clean several objects without having to select first.

**Comment by JamesDeAntonis** [comment]
Is there still no eraser here?  I would also like to use one

**Comment by rsmyst** [comment]
Hijacking thread to say an eraser would go a long way in helping in making the switch for me. i use excalidraw through the obsidian plugin which has been really handy but would be better if an actual eraser was added. good luck

**Comment by TheMathGuyd** [comment]
Same. 8 months after my first comment and I am still looking for an embeddable drawing solution that has an eraser. I still use one tool for stylus work, and then vectorize it in another program, and then preview it in another. A drawing program should have an eraser. The question is: is ExcaliDRAW meant to be a drawing program?

**Comment by dwelle** [comment]
Hey everyone. We will be adding an eraser — well sort of. While we would love to have a proper erasing functionality, due to how Excalidraw works right now (including the freedraw tool), what we will start with is just a tool that deletes objects under cursor.

https://github.com/excalidraw/excalidraw/issues/2052

**Comment by rsmyst** [comment]
@AdamL2021 
The latest update in the obsidian plugin has blessed us with a thin width pen mode. It's absolutely useful for noting down finally instead of looking like a splotch on the board.

**Comment by charlescohanlon** [comment]
Any chance we're ever going to see stylus-activated eraser support? That's the one thing barring me from switching to Obsidian w/ Excalidraw. I'm using the Samsung Tab s7+
